coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
+                return False
```

Comment:
didn't meant to be mean btw. but this bug is so important that I'm pushing a fix right now, I hope you don't mind. I'm mentioing you as co author in the commit message because I took some code from this PR, esp the super with the cls saved me loads of time